### DISCUSSION TITLE
BatchCommand

### DISCUSSION THREAD

**Discussion by bobskigit** [ravendb/ravendb#13989] [Q&A] [answered]
In the ravenDB documentation there is this example

var batch = new BatchCommand(documentStore.Conventions, session.Advanced.Context, commands);
    await session.Advanced.RequestExecutor.ExecuteAsync(batch, session.Advanced.Context);

For the life of me I cannot find BatchCommand in any of the raven libs

Any ideas?

**Comment by ppekrol** [comment, marked as answer]
Hi, it was renamed to `SingleNodeBatchCommand`.
